### ISSUE TITLE
cross-build-start / cross-build-end doesn't match if full path is used

### ISSUE THREAD

**Issue by zrzka** [resin-io-projects/armv7hf-debian-qemu#16]
## Description

See this [Front message](https://app.frontapp.com/open/msg_25zi3mr) for more details.

TL;DR;

User has `RUN [ "/usr/bin/cross-build-end" ]` in the `Dockerfile` (and same for `-start`). When the full path was removed (`RUN [ "cross-build-end" ]`), issue was fixed.

## Test case

```go
package main
import (
	"fmt"
	"os"
)

func main() {
  fmt.Println(os.Args[0])
}
```

Built, moved to `~/bin`.

```
 ~ zrzka
zrzka
 ~ ~/bin/zrzka 
/Users/robertvojta/bin/zrzka
 ~ 
```

It's caused by [this](https://github.com/balena-io-playground/armv7hf-debian-qemu/blob/master/resin-xbuild.go#L57) and [this](https://github.com/balena-io-playground/armv7hf-debian-qemu/blob/master/resin-xbuild.go#L59) case statement.

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_g1b7)
